<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSROutboundEditor</class>
 <widget class="QWidget" name="SSROutboundEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="ssrPasswordTxt"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Method</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="ssrMethodCombo">
     <item>
      <property name="text">
       <string>none</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">rc4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">rc4-md5-6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">rc4-md5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">aes-128-cfb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">aes-192-cfb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">aes-256-cfb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">aes-128-ctr</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">aes-192-ctr</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">aes-256-ctr</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">camellia-128-cfb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">camellia-192-cfb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">camellia-256-cfb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">bf-cfb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">cast5-cfb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">des-cfb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">idea-cfb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">rc2-cfb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">seed-cfb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">salsa20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">chacha20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">chacha20-ietf</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Protocol</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="ssrProtocolCombo">
     <item>
      <property name="text">
       <string notr="true">origin</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">auth_sha1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">auth_sha1_v2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">auth_sha1_v4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">auth_aes128_sha1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">auth_aes128_md5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">auth_chain_a</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">auth_chain_b</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>auth_chain_c</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>auth_chain_d</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>auth_chain_e</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>auth_chain_f</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Obfuscator</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="ssrObfsCombo">
     <item>
      <property name="text">
       <string notr="true">plain</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">http_simple</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">http_post</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">tls1.2_ticket_auth</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Protocol Params</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="ssrProtocolParamsTxt"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Obfs Params</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="ssrObfsParamsTxt"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
